src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -119,6 +119,7 @@
 
           <button v-if="isEditable"
                   type="button"
+                  v-b-tooltip.hover="transIf(activeBlockId,'flow-builder.tooltip-duplicate-block')"
```

Comment:
Rather than the indirection of `transIf()`, could we use the built-in `disabled` configuration property? Like:
```
:v-b-tooltip.hover={title: trans('flow-builder.tooltip-duplicate-block'), disabled: activeBlockId}"
```

Reply:
@bzabos , we're already using the `disabled` prop.
But in this particular scenario, if we delete the block, the delete tooltip remains open.
<img width="266" alt="Screen Shot 2021-03-18 at 2 27 17 PM" src="https://user-images.githubusercontent.com/68745918/111696294-c5331d80-87f9-11eb-9d3a-9586c5913147.png">

Developers have reported this: https://github.com/bootstrap-vue/bootstrap-vue/issues/1892

I will implement the suggested solution then: `Place your tooltip on an outer wrapper element, rather than the disabled button element.` https://github.com/bootstrap-vue/bootstrap-vue/issues/1892#issuecomment-502703464

But this means, the user will still see the tooltip when making a mouse over

Reply:
But to disable tooltip on block side when we are on **View Mode**, the above solution won't work. Eg: https://github.com/FLOIP/flow-builder/pull/42/files#diff-43f7c5f1a1252fa694baadbe4e2a0ec2f48c3a9041968f25ed1cfab7b4bdff9aR102

I think, I'm keeping the `transIf()` workaround for now, unless you have another solution.